src/renderer/components/+nodes/nodes.scss

```diff
@@ -37,7 +37,7 @@
       align-self: center;
 
       .LineProgress {
-        color: $lensBlue;
```

Comment:
Were `lensBlue` and `lensMagenta` defined somewhere?

Reply:
Nope, they where defined in linking sass-vars to css-vars in deleted `theme-vars.scss`:

```scss
$lensBlue: var(--blue);
```